src/jobs/BenchmarkJob.jl

```diff
@@ -226,20 +221,21 @@ function Base.run(job::BenchmarkJob)
         nodelog(cfg, node, "finished primary build for $(summary(job))")
 
         # run the comparison job (or if it's a daily job, gather results to compare against)
-        if job.isdaily # get results from previous day (if it exists, check the past 120 days)
+        if job.isdaily # get results from previous day (if it doesn't exists, check the past 31 days)
             try
                 nodelog(cfg, node, "retrieving results from previous daily build")
                 found_previous_date = false
                 i = 1
-                while !found_previous_date && i < 121
+                while !found_previous_date && i < 31
```

Comment:
we sometimes have had issues where the node is down for maintenance for a month (because of TIG, and me), so 120 was better

Reply:
Ah, true, that does address this. Okay then